ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -64,6 +65,12 @@
       + "will not start serving requests until all preconditions are satisfied. Until that time, "
       + "requests will return a 503 error";
 
+  static final String KSQL_SERVER_ERRORS =
+          KSQL_CONFIG_PREFIX + "server.errors";
+  private static final String KSQL_SERVER_ERRORS_DOC =
+          "A class implementing Errors. This allows the KSQL server to return pluggable "
```

Comment:
wrong interface. I'd also advice changing to something like:

```suggestion
          "A class the implementing " + ErrorMessages.class.getSimpleName() + " interface. This allows the KSQL server to return pluggable "
```

So that the type itself is referenced in the creation of the doc text. This has the benefit that it will automatically be changed if someone changes the name of the class, and people will find this bit of code if/when using the IDEs features to find where the class is used.